adapters/adview/adview.go

```diff
@@ -157,3 +178,44 @@ func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType,
 	}
 	return mediaType, nil
 }
+
+func getMediaTypeForBid(bid openrtb2.Bid, reqImps []openrtb2.Imp) (openrtb_ext.BidType, error) {
+	mediaType := openrtb_ext.BidTypeBanner
+	for _, imp := range reqImps {
+		if bid.ImpID == imp.ID {
+			if bid.Ext != nil {
+				var bidExt *adviewBidExt
+				err := json.Unmarshal(bid.Ext, &bidExt)
+				if err == nil {
+					return getBidType(bidExt), nil
+				} else {
+					return mediaType, err
+				}
+			}
+			return mediaType, nil
+		}
+	}
+	return mediaType, nil
+}
+
+// getBidType returns the bid type specified in the response bid.ext
+func getBidType(bidExt *adviewBidExt) openrtb_ext.BidType {
+	// setting "banner" as the default bid type
+	bidType := openrtb_ext.BidTypeBanner
+	if bidExt != nil {
+		switch bidExt.BidType {
+		case 0:
+			bidType = openrtb_ext.BidTypeBanner
+			//fmt.Println("==== [adview] getBidType(): ==> banner ===")
+		case 1:
+			bidType = openrtb_ext.BidTypeVideo
+			//fmt.Println("==== [adview] getBidType(): ==> video ===")
```

Comment:
seems debug left over. should remove commented fmt lines on line 209 and 212

Reply:
removed.